### PR TITLE
Landing-Page

### PR DESCRIPTION
**Pull request by lomesshh**
- In this PR I've added
-- Homepage for video library
-- Login & Signup page with formik including data handling
-- profile page
-- components --> Navbar & Footer

### CODE REVIEW

File: src/frontend/stylesheets/Auth.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  margin-bottom: 3.5rem;
+}
+
+.error__display {
+  color: red;
+}
+
+.matched__display {
+  color: green;
+}
```
Comment (anuj-kumary) [inline src/frontend/stylesheets/Auth.css]: try to avoid using hardcode color value and also use color name as hex value

**Review by anuj-kumary — APPROVED** [review]
everything looks good to me good use of naming conventions

File: src/frontend/stylesheets/Auth.css (hunk 1 above)
Comment (lomesshh) [reply src/frontend/stylesheets/Auth.css]: as of now, I am using these colors just once that's why I didn't create a separate variables for them
I've changed those values to hex code

**Review by Hnh013 — APPROVED** [review]
the UI looks good, balanced theme colors , clean and structured code . Approving changes

**Review by Aman-Jain1996 — APPROVED** [review]
Code structure looks clean and good to merge

**Pull request merged**